### ISSUE TITLE
[E4] Duda sobre la neutralización de follow up

### ISSUE THREAD

**Issue by martina-abello** [franciscogazitua/IIC2113-2024#210]
los  efectos de neutralización de follow up afectan si la unidad tiene follow up garantizado, o sólo la condición de Spd?

En el test 000 Mix, se tiene que 
![imagen](https://github.com/user-attachments/assets/2db1efe0-085e-45c5-854e-709cad11e95b)
ninguna puede hacer follow-up, aunque Ike tenga garantización de follow up. esto me llevaría a asumir que la negación de follow up afecta las garantizaciones. 

sin embargo, en el test 011 Mix, se tiene que
![imagen](https://github.com/user-attachments/assets/7e208ea7-94e0-40a3-bca8-a39aefd7735b)
Eirika puede hacer follow up, pese a tener una neutralización de follow up activa. así que no entiendo la relación.

gracias de antemano

**Comment by cahinostroza** [comment]
Efectivamente, las neutralizaciones afectan las garantizaciones y viseversa.
Puedes pensar que cada unidad tiene una especie de contador de follow-ups, cuando se cumple la condición de velocidades y cuando hay una garantización, el contador aumenta, y cuando hay una neutralización el contador disminuye.
Por eso Lyn no hace un follow up en el primer caso (+1 condición de velocidad, -1 neutralización) y Eirika si hace un follow-up (+1 condición de velocidad, +1 garantización, -1 neutralización).
